content/compose/profiles.md

```diff
@@ -83,7 +83,7 @@ $ docker compose --profile frontend --profile debug up
 ```
 
 ```console
-$ COMPOSE_PROFILES=frontend,debug docker compose up
+$ COMPOSE_PROFILES=frontend,debug
```

Comment:
Looking at this example, I think this was intentional; this line does 2 things;

- set the `COMPOSE_PROFILES` env-var to `frontend,debug` (for the duration of the command)
- execute `docker compose up` (which now enables the `frontend` and `debug` profiles)

Reply:
Perhaps the docs could describe that though, and explain why the env-var is used like this (commonly this is something when you want a setting to only be used for a single command, without making the change permanent)